tests/store/test_store_api.py

```diff
@@ -1,8 +1,12 @@
 """Tests for the store api"""
 import datetime
+from typing import List
 
+from housekeeper.store.models import File
 
-def test_fetch_bundles(populated_store, bundle_data_old):
+
+
+def test_fetch_bundles(populated_store, bundle_data_old, time_stamp_now):
```

Comment:
```suggestion
def test_get_files_before(populated_store, bundle_data_old, time_stamp_now):
```
seems more appropriate